File: fixtures/ts/src/service.ts
import { Greeter, helper } from './util.ts'

export class Service {
  run(): string {
    const greeter = new Greeter('world')
    return `${greeter.greet()} / ${helper('direct')}`
  }
}

export function unused(): void {}

export const defaultService = new Service()
export const CONFIG = { retries: 3 }

File: fixtures/py/pkg/service.py
from .util import Greeter, helper


def run():
    greeter = Greeter("world")
    return greeter.greet() + helper("direct")


# Module-level object built by a call — the structure of many declarative files.
default_greeter = Greeter("module-level")
_private_thing = Greeter("hidden")
PLAIN_CONSTANT = 42
